driver/portfolio_runner.py

```diff
@@ -83,11 +83,12 @@ def compute_run_time(timeout, configs, pos):
     print("remaining time: {}".format(remaining_time))
     relative_time = configs[pos][0]
     remaining_relative_time = sum(config[0] for config in configs[pos:])
-    print("config {}: relative time {}, remaining {}".format(
-          pos, relative_time, remaining_relative_time))
     # For the last config we have relative_time == remaining_relative_time, so
     # we use all of the remaining time at the end.
-    return remaining_time * relative_time / remaining_relative_time
+    limit = int(remaining_time * relative_time / remaining_relative_time)
```

Comment:
I wonder if we should move this computation into a function, which can then document what we're doing there including the fractional-second-resolution to second-resolution step that happens here.

Also, perhaps it would be better to introduce a "plus epsilon" thing here like we do for heuristics. For example, this might be unexpected:

```
>>> int(1. / 49 * 49)
0```

Reply:
I moved the rounding (+epsilon) into a function and added the documentation.